driver-core/src/main/com/mongodb/DuplicateKeyException.java

```diff
@@ -19,9 +19,21 @@
 import org.bson.BsonDocument;
 
 /**
- * Subclass of {@link WriteConcernException} representing a duplicate key exception
+ * A subclass of {@link WriteConcernException} representing a duplicate key exception
```

Comment:
Nit: end the sentence with a period?

Reply:
Sure, though it's technically not a complete English sentence. :)

Reply:
Done